<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>701</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">    QStackedWidget {
        background-color: lightblue; /* Replace 'lightblue' with your desired color */
    }</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="startScreen">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #6A0DAD; /* Purple */
    font-size: 32px;
    font-weight: bold;
    font-family: &quot;Arial Black&quot;;
    background-color: transparent;
    padding: 10px;
    border: 2px solid #BA55D3; /* Orchid border */
    border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Crack N Clash</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>311</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff6f61;
    color: #fff5e1;
}
</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff6f61;
    color: #fff5e1;
}
</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="difficultyScreen">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #0077b6;
    font-size: 32px;
    font-weight: bold;
    font-family: &quot;Arial Black&quot;;
    background-color: transparent;
    padding: 10px;
    border: 2px solid #00b4d8;
    border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Select Difficulty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>181</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="easyButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff6f61;
    color: #fff5e1;
}
</string>
         </property>
         <property name="text">
          <string>Easy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediumButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff6f61;
    color: #fff5e1;
}
</string>
         </property>
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hardButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff6f61;
    color: #fff5e1;
}
</string>
         </property>
         <property name="text">
          <string>Hard</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gameScreen">
     <widget class="QTextEdit" name="outputTextEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="timeBar">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>460</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>380</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>420</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
